### PR TITLE
Added routing, page with and without left-sidebar

### PR DESCRIPTION
**Pull request by Setup1337**
image
image
image
After confirmation email, it's redirects to organizations pages
In a future will fix all bugs and discuss our user flow in meetings

### CODE REVIEW

File: backend/Tasque.Core.WebAPI/Controllers/OrganizationController.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public class OrganizationController : EntityController<Organization, Organizatio
         [HttpGet]
         public virtual IActionResult GetUserOrganizationsById(int userId)
         {
-            var organizations = _service.GetUserOrganizations(userId);
+            var organizations = _service.GetUserOrganizations(userId).Result;
```
Comment (oleksandr-vovk) [inline backend/Tasque.Core.WebAPI/Controllers/OrganizationController.cs]: Should make method async and use await here

**Review by oleksandr-vovk — CHANGES_REQUESTED** [review]

File: backend/Tasque.Core.WebAPI/Controllers/OrganizationController.cs (hunk 1 above)
Comment (Setup1337) [reply backend/Tasque.Core.WebAPI/Controllers/OrganizationController.cs]: Done.

File: frontend/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+    ]
+  },
+  { path: 'project', component: PageWithSidebarComponent,
+    children: [
+      { path: 'board', component: TasqueBoardComponent },
+      { path: 'team', component: TasqueTeamComponent },
+      { path: 'settings', component: TasqueProjectSettingsComponent },
+    ]
+  },
+  { path: '**', redirectTo: '/auth/login', pathMatch: 'full' },
```
Comment (ihor-il) [inline frontend/src/app/app-routing.module.ts:29]: `/auth` automatically redirects to `/auth/login` so there's no need to be so explicit

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html [hunk 3]
```diff
@@ -10,39 +10,39 @@
     <ul>
       <li>
         <div class="menu-item">
-          <a href="/board">
+          <a href="project/board">
```
Comment (ihor-il) [inline frontend/src/shared/components/left-sidebar/left-sidebar.component.html]: could use `routerLink` instead of `href` for consistency

**Review by ihor-il — DISMISSED** [review]

**Review by oleksandr-vovk — DISMISSED** [review]

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html (hunk 3 above)
Comment (Anatolii-Ivanov) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.html]: Valuable  comment
Href is the basic attribute provided by Html to navigate through pages which reloads the page on click.
routerLink is the attribute provided by angular to navigate to different components without reloading the page.
https://stackoverflow.com/questions/58081360/anchor-href-vs-angular-routerlink#:~:text=Href%20is%20the%20basic%20attribute,components%20without%20reloading%20the%20page.

File: frontend/src/app/auth/components/confirm-email-page/confirm-email-page.component.ts [hunk 4]
```diff
@@ -32,6 +32,10 @@ export class ConfirmEmailPageComponent implements OnInit {
         this.notifService.success(
           'Email confirmed. You will be redirected to your profile',
         );
+        this.router.navigate(['../..', 'organizations'], {
+          replaceUrl: true,
+          relativeTo: this.route,
+        });
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/auth/components/confirm-email-page/confirm-email-page.component.ts:40]: Do we also redirect to organizations page after login?

File: frontend/src/shared/components/left-sidebar/left-sidebar.component.html (hunk 3 above)
Comment (Setup1337) [reply frontend/src/shared/components/left-sidebar/left-sidebar.component.html]: Changed href to router link

File: frontend/src/app/auth/components/confirm-email-page/confirm-email-page.component.ts (hunk 4 above)
Comment (Setup1337) [reply frontend/src/app/auth/components/confirm-email-page/confirm-email-page.component.ts:40]: Added redirection to organizations page after login

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**